web-crawler/src/main/java/org/iqu/webcrawler/news/management/NewsEndpoint.java

```diff
@@ -112,16 +110,22 @@ public Response retrieveSource() {
 		int status;
 		String response = "";
 		status = 0;
-		Source source = new Source("1", "BNR Brasov", "This is the official BNR site");
+		Set<Source> sources = new HashSet<Source>();
 
-		if (source.getDisplayName().equals("BNR Brasov")) {
-			status = 200;
-			return Response.status(status).entity(source).build();
-		} else {
+		// ToDo get sources from db
+		sources.add(new Source("1", "BNR Brasov", "This is the official BNR site"));
+		sources.add(new Source("2", "BNR Brasov", "This is the official BNR site"));
+		sources.add(new Source("3", "BNR Brasov", "This is the official BNR site"));
+
+		if (sources.isEmpty()) {
 			status = 404;
 			response = "\"error\" : \"Could not fetch sources, please try again later.\"";
 			return Response.status(status).entity(response).build();
 		}
+
+		status = 200;
+		return Response.status(status).entity("{\"sources\" : " + "\"" + sources + "\"}").build();
```

Comment:
Use an entity Sources that will have as a data member a set of sources and thus so you won't need to hardcode the sources keyword here.